.github/workflows/publish.yml

```diff
@@ -4,6 +4,11 @@ on:
   release:
     types: [created]
 
+permissions:
+  contents: read
+  packages: write
```

Comment:
Do we need `packages` here? I don't think we do given we publish only to `npm` now

Reply:
I think `contents: read` is enough

Reply:
Thanks for your comments. I updated the PR.